src/main/app/fact/FactApi.ts

```diff
@@ -12,19 +12,26 @@ export class FactApi {
     return this.axios
       .get('/courts/all')
       .then(results => results.data)
-      .catch(err => {
-        this.logger.error(err);
-        return [];
-      });
+      .catch(this.errorHandler([]));
   }
 
   public getCourt(slug: string): Promise<{}> {
     return this.axios
       .get(`/courts/${slug}`)
       .then(results => results.data)
-      .catch(err => {
-        this.logger.error(err);
-        return {};
-      });
+      .catch(this.errorHandler({}));
```

Comment:
why an object here and an array on line 15?

Reply:
it's what it was before, I assume because the search returns a list of courts and that request returns a single court